### PR TITLE
Fix ComboBox accessibility attributes

### LINKED ISSUE #4450: [Accessibility] ComboBox has invalid aria-* attributes
### Bug Report

- __Package version(s)__: 5.78
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

#### Describe the issue:

* role="option" widget has invalid attribute aria-pressed
* role="combobox" has invalid aria attributes

Refer to https://www.w3.org/TR/wai-aria-1.1/#combobox

### PR DESCRIPTION
**Pull request by stephenzsy**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #4450
- [x] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by dzearing** [comment]
@stephenzsy I'm having a difficult time helping to resolve this conflict. Could you pull master into your branch?

**Comment by stephenzsy** [comment]
I just did rebase. Please review

**Comment by dzearing** [comment]
@jspurlin Can you please sign off? Thank you!

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx [hunk 1]
```diff
@@ -376,17 +384,16 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
...
+                role='textbox'
                 aria-readonly={ ((allowFreeform || disabled) ? undefined : 'true') }
                 readOnly={ disabled || !allowFreeform }
                 aria-labelledby={ (label && (id + '-label')) }
                 aria-label={ ((ariaLabel && !label) ? ariaLabel : undefined) }
                 aria-describedby={ describedBy + (keytipAttributes['aria-describedby'] || '') }
                 aria-activedescendant={ this._getAriaActiveDescentValue() }
                 aria-disabled={ disabled }
-                aria-owns={ (id + '-list') }
+                aria-controls={ isOpen ? (id + '-list') : undefined }
```
Comment (jspurlin) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx]: All of these changes are new patterns in aria 1.1. And the spec states that aria 1.0 patterns should still continue to be supported. We should verify that this reads out correctly in latest Edge and Narrator (at least) before making this change. I feel like this is making the code work for the widest array of browser/screen reader combos and we should only make this change once this pattern is widely supported

**Comment by stephenzsy** [comment]
The current implementation fails validation in Chrome DevTools

File: packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx [hunk 2]
```diff
@@ -176,8 +179,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         'aria-label': ariaLabel,
         'aria-labelledby': ariaLabelledBy,
         'aria-describedby': ariaDescribedBy,
-        'data-is-focusable': ((this.props as any)['data-is-focusable'] === false || disabled || this._isSplitButton) ? false : true,
-        'aria-pressed': checked
```
[the remark this answers is not in the document]
Comment (stephenzsy) [reply packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx]: It will get stipped if it is specified as undefined.

**Comment by jspurlin** [comment]
@stephenzsy let's check the end result on a combination of browser/screen readers (especially with Edge/Narrator). From the aria 1.1 spec, the aria 1.0 pattern is still valid and if the result up updating to the aria 1.1 pattern is negative we shouldn't update at this point

**Comment by dzearing** [comment]
This PR seems to have lost traction as it is 61 days old. @stephenzsy can you and @jspurlin talk over mail about the right change here? I'm going to close this for now, as this PR needs updating due to the conflicts, but please discuss the correct change and then resubmit the fix.

**Pull request closed without merging**